Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -94,6 +96,12 @@ void vtkMRMLClipModelsNode::ReadXMLAttributes(const char** atts)
       ss << attValue;
       ss >> ClipType;
       }
+    else if (!strcmp(attName, "clippingMethod"))
+      {
+      std::stringstream ss;
+      ss << attValue;
+      ss >> ClippingMethod;
```

Comment:
Sure!  Would you like me to fix this for the clipState/Type variables as well? They have a similar enum structure.